### PR TITLE
crimson/tools: add fio metrics support

### PR DESCRIPTION
**Pull request by liu-chunmei**
Signed-off-by: chunmei-liu <chunmei.liu@intel.com>

### CODE REVIEW

File: tools/crimson/seastore_metrics_analyze.py [hunk 1]
```diff
@@ -52,6 +52,11 @@ def parse_bench_file(bench_file):
                 writes = int([x for x in line.split(' ') if x][3])
             elif line.startswith("Object size:"):
                 obj_size = int([x for x in line.split(' ') if x][2])
+            elif line.startswith("     issued rwts: total"):
```
Comment (cyx1231st) [inline tools/crimson/seastore_metrics_analyze.py:55]: Seems some fio outputs only contains "issued rwt: total". see https://gist.github.com/liu-chunmei/f696b9c4f31b123fb223cdd47f13c8ea#crimson-4-kb-random-write-two-fio-job-number-instances

File: tools/crimson/README.rst [hunk 2]
```diff
@@ -18,5 +18,40 @@ To start the test envionment:
...
+rbdname=fio_test
+rw=randwrite
+bs=4K
+runtime=60
+numjobs=1
+direct=1
+group_reporting
+
+[rbd_iodepth32]
+iodepth=2
```
Comment (cyx1231st) [inline tools/crimson/README.rst:54]: This doesn't format well in the representation, add to `.. code-block:: console`

File: tools/crimson/README.rst [hunk 3]
```diff
@@ -18,5 +18,40 @@ To start the test envionment:
...
+
+
+
```
Comment (cyx1231st) [inline tools/crimson/README.rst:57]: Remove empty lines.

**Comment by liu-chunmei** [comment]
@cyx1231st updated.

File: tools/crimson/seastore_metrics_analyze.py [hunk 4]
```diff
@@ -52,7 +52,7 @@ def parse_bench_file(bench_file):
                 writes = int([x for x in line.split(' ') if x][3])
             elif line.startswith("Object size:"):
                 obj_size = int([x for x in line.split(' ') if x][2])
-            elif line.startswith("     issued rwts: total"):
+            elif line.startswith("     issued"):
```
Comment (cyx1231st) [inline tools/crimson/seastore_metrics_analyze.py:55]: Should be in commit https://github.com/ceph/cbt/pull/239/commits/1d19209890038cbed763429aaec41ecdad636307

**Review by cyx1231st — APPROVED** [review]
LGTM except some nits.

File: tools/crimson/seastore_metrics_analyze.py (hunk 4 above)
Comment (liu-chunmei) [reply tools/crimson/seastore_metrics_analyze.py:55]: fixed.

**Pull request merged**